src/components/CreateAnalysis/CreateAnalysisDetail.tsx

```diff
@@ -74,14 +77,26 @@ const CreateAnalysisDetail: React.FC<CreateAnalysisDetailProps> = (props) => {
             </div>
           </div>
         </div>
-        <div className="detail-bottom-wrapper">
-          <div className="detail-select-studies">
-            {studyInstances.map((study: StudyInstance, i) => (
-              <SelectionStudy key={i} {...study}></SelectionStudy>
-            ))}
-          </div>
-          <SelectedStudyDetail></SelectedStudyDetail>
-        </div>
+        <PageSection className="PatientLookupWrapper" variant={PageSectionVariants.light}>
```

Comment:
How come you're referencing PatternFly PageSection here?

Reply:
I used PageSection here to format the layout of my FileLookup component. Other components like PatientLookup use this as well, so I decided to use it here for consistency.